### ISSUE TITLE
Problems installing the latest version (3.2.0.3) through the Plugin Gallery in Open Journal Systems (3.3.0-8)

### ISSUE THREAD

**Issue by sbirAECA** [NateWr/bootstrap3#176]
Nate, good afternoon, I'm Carlos Martinez and I have a question for you regarding the Boostrap3 theme, when I install the latest version (3.2.0.3) through the Plugin Gallery in Open Journal Systems (3.3.0-8), in the directory that contains the plugin a '.git' directory is added. I am controlling the new installation via git and when I create a new commit that includes your plugin I am getting the following message:
"warning: adding embedded git repository: plugins/themes/bootstrap3
hint: You've added another git repository inside your current repository.

In short, you can't add a repository inside another repository. Git is recognizing your code as a repository.
I don't know if this is quite right, but so far I haven't found the '.git' directory every time I install a plugin in OJS.

**Comment by sbirAECA** [comment]
Nate, forget about last issue, I have found a solution

**Comment by NateWr** [comment]
Thanks for reporting this, @sbirAECA. The `.git` directory shouldn't have been included. I've just uploaded a new release without this directory, and it should be available in the plugin gallery soon.

**Comment by asmecher** [comment]
@NateWr, are you using the PKP plugin release tool to build this plugin? It should already be excluding `.git`: https://github.com/pkp/pkp-plugin-cli/blob/main/src/release/buildRelease.js#L99

**Comment by NateWr** [comment]
Yes, I did today. I think the older release package must have been built from an earlier version of the release tool.

**Comment by sbirAECA** [comment]
@asmecher, good morning,
I'm customising an OJS installation from scratch, and I'm likely to keep downloading plugins from the 'Plugin Gallery'.
Do you want me to report to you every time I encounter a similar problem related to the 'git' or 'github' directories?
Regards
Carlos

**Comment by NateWr** [comment]
Sure! We don't set rules on third-party plugins, but I think most maintainers (including us) would like to know if we've accidentally included that in a release package.
